src/mixins/README.md

```diff
@@ -0,0 +1,128 @@
+# Mixins
+
+This directory contains source files for the mixins exported by this extension.
+
+## Overview
+
+Sometimes it's helpful to write partial classes and then combining them together to build more powerful classes. This pattern is called Mixins (mixing in partial classes) and is supported by LoopBack 4. 
+
+LoopBack 4 supports mixins at an `Application` level. Your partial class can then be mixed into the `Application` class. A mixin class can modify or override existing methods of the class or add new ones! It is also possible to mixin multiple classes together as needed.
+
+### High level example
+```ts
+const MixedClass = MyMixinClass(Application);
+const MultipleMixedClassesClass = MyMixinClass(MyMixinClass2(Application));
```

Comment:
Could you please change one of these examples to show how a mixin can be used with class inheritance? E.g.

```ts
class MixedApp extends MyMixinClass(Application) {
}
```

As I commented earlier, I think this is the usage we should be promoting, and it may not be obvious to all TypeScript developers that mixins can be invoked inside `extends` clause.